docs/events.md

```diff
@@ -4,63 +4,60 @@ linkTitle: "Events"
 weight: 2
 ---
 -->
-# Events
+# Events in Tekton
 
-Tekton runtime resources, specifically `TaskRuns` and `PipelineRuns`,
-emit events when they are executed, so that users can monitor their lifecycle
-and react to it.
+`TaskRuns` and `PipelineRuns` emit [Kubernetes events](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#event-v1-core)
+during their execution. This allows you to monitor and react to what's happening during execution by
+retrieving those events using the `kubectl describe` command. Tekton can also emit [CloudEvents](https://github.com/cloudevents/spec).
```

Comment:
Given we explicitly cover `kubectl describe` usage in the next paragraph as well I wonder if we need to mention it here?  This paragraph appears more like a summary of the kinds of events tekton supports.  What do you think something like this:

> `TaskRuns` and `PipelineRuns` emit both [Kubernetes events](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#event-v1-core) and, optionally, [CloudEvents](https://github.com/cloudevents/spec). This allows you to monitor and react to what's happening during execution.

Reply:
I am never doing rewrites late in the day again.  This paragraph is a rewrite of the one that follows, which should have been deleted. And now has been. Sorry about that!